services/hdbpp-ttl/setup/hdbpp_ttl.conf

```diff
@@ -19,4 +19,14 @@ hdb-cluster:
         port: 5432
 
         # if node defined, if not set assumes a default of 'hdb'
-        database: hdb
\ No newline at end of file
+        database: hdb
+
+    # report ttl processing results to the following defined rest end point
+    rest_endpoint:
+
+        # enable reporting to the end point, if not set assumes a default of default false
+        enable: false
+
+        # api url to contact to send ttl results to, if not set assumes a default of default http://localhost:10666/api/v1
+        # edit if the rest api is on a different host/port
+        api_url: http://localhost:10666/api/v1
```

Comment:
shouldn't we add the endpoint entry as well?

Reply:
Removed it on purpose, since I do not see us changing it in future. Adding it only promotes the idea it is mutable, when its not.